jb/itch5/order_book_depth.hpp

```diff
@@ -153,9 +177,10 @@ class order_book {
 
   buys buy_;
   sells sell_;
+  book_depth_t book_depth_ = 0;       
```

Comment:
Just curious, what is the time complexity of querying the size of the buy_ and sell_ data structures?  If it is constant, then keeping a separate variable is not really that much of an optimization, right?